lib/lte_link_control/lte_lc.c

```diff
@@ -504,7 +504,7 @@ static int w_lte_lc_init(void)
 	}
 
 #if defined(CONFIG_LWM2M_CARRIER) && !defined(CONFIG_GPS_USE_SIM) && \
-    (defined(CONFIG_BOARD_NRF9160_PCA20035NS) || \
+    (defined(CONFIG_BOARD_THINGY91_NRF9160NS) || \
      defined(CONFIG_BOARD_NRF9160_PCA10090NS))
```

Comment:
The below line (and similar elsewhere) should also be changed at some point. Probably not in this PR though